recipes/url-normalize/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "url-normalize" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d23d3a070ac52a67b83a1c59a0e68f8608d1cd538783b401bc9de2c0fac999b2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5
```

Comment:
conda-forge isn't supporting 2.7. So unless you explicitly need 2.7 support for this package, best to just change this to `>=3.6` here and below.

Reply:
Oh, well that's a relief! I thought I might be obligated to include 2.7 since the project maintainer included it in their package metadata. Just updated that to `>=3.6`.